### DISCUSSION TITLE
Question about GPTQ models and running GGML models

### DISCUSSION THREAD

**Discussion by LazyAIEnjoyer** [PromtEngineer/localGPT#187] [General]
Are GPTQ models only for running on GPU? Because when I try to run it with device-type cpu, it asks for a gpu. Is it also possible to add GGML model functionality?

**Comment by KonradHoeffner** [comment]
1. Yes, you can only use GPTQ  on a GPU.
2. Yes, GGML is supported as well, which runs on CPU and also on GPU.
